torch/_dynamo/utils.py

```diff
@@ -1267,7 +1267,7 @@ def get_fake_value(node, tx):
 
     def fake_wrapper(e):
         if isinstance(e, torch.Tensor):
-            assert isinstance(e, FakeTensor)
+            assert is_fake(e)
```

Comment:
if this PR is held up in code review, you should send in these changes on their own